mayan/apps/documents/models/metric.py

```diff
@@ -4,3 +4,4 @@
 class Metric(models.Model):
     """Model that describes a metric"""
     metric_name = models.TextField()
+    metric_type = models.TextField()
```

Comment:
Got it. Thanks for adding this.

However, I thought we only allow metrics with integer values in a range (like 1~5)? I was thinking that we should use [ChoiceField](https://docs.djangoproject.com/en/3.2/ref/forms/fields/#choicefield) for all values in the review form, so that we restrict the user to only select from a fixed set of integer values (e.g. {1, 2, 3, 4, 5}) for each metric. Just like what @shawnchan2014 did [here](https://github.com/CMU-313/fall-2021-hw2-connect5/pull/24/commits/14c737ef224b9ca897ef25e3bfda4aba10b8e9c8#diff-cdbe9abc3688aecd46fe788ff91da1899c801dff0bb0d1a4891a108a6f54a532R44)

Anyway, we can keep this `metric_type` field in case we need it in the future.

Reply:
Nevermind, I just saw your comment [here](https://github.com/CMU-313/fall-2021-hw2-connect5/pull/24#pullrequestreview-763700800).